server/lomas_server/data_connector/factory.py

```diff
@@ -2,18 +2,18 @@
 
 from admin_database.admin_database import AdminDatabase
 from constants import PrivateDatabaseType
-from private_dataset.path_dataset import PathDataset
-from private_dataset.private_dataset import PrivateDataset
-from private_dataset.s3_dataset import S3Dataset
+from data_connector.data_connector import DataConnector
+from data_connector.path_dataset import PathDataset
```

Comment:
Should it now be s3_connector and path_connector?

Reply:
yes, indeed! I'll change now